### PR TITLE
Add Flake8 linting to Pants configuration

### PR DESCRIPTION
**Pull request by christophermaier**
Initially checks for unused imports, since we have a fair number of
those. More checks can be added in the future.

Signed-off-by: Christopher Maier <chris@graplsecurity.com>

### CODE REVIEW

File: src/python/grapl_analyzerlib/grapl_analyzerlib/viewable.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 LOGGER.addHandler(logging.StreamHandler(stream=sys.stdout))
 
 V = TypeVar("V", bound="Viewable")
-Q = TypeVar("Q", bound="Queryable")
+Q = TypeVar("Q", bound=Queryable)
```
Comment (christophermaier) [inline src/python/grapl_analyzerlib/grapl_analyzerlib/viewable.py:37]: Highlighting this for reviewers.

File: src/python/grapl_analyzerlib/grapl_analyzerlib/queryable.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
     from grapl_analyzerlib.viewable import Viewable
 
 Q = TypeVar("Q", bound="Queryable")
-V = TypeVar("V", bound="Viewable")
+V = TypeVar("V", bound=Viewable)
```
Comment (christophermaier) [inline src/python/grapl_analyzerlib/grapl_analyzerlib/queryable.py:36]: Highlighting this for reviewers.

File: .flake8 [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    F401
+ignore =
+    # E501: line too long
+    E501
+per-file-ignores =
+    # This file is basically *all* imported-but-unused
+    src/python/grapl_analyzerlib/grapl_analyzerlib/prelude.py: F401
+exclude =
+    # This is apparently a generated file
+    src/python/grapl_analyzerlib/grapl_analyzerlib/graph_description_pb2.pyi
```
Comment (jgrillo-grapl) [inline .flake8:14]: 🤢 this file seems like it shouldn't be here... We generate all our proto stubs at build time. Not that it's something we need to address in this PR.

**Review by jgrillo-grapl — APPROVED** [review]
LGTM

**Review by jgrillo-grapl — APPROVED** [review]

**Pull request merged**